package.json

```diff
@@ -25,8 +25,8 @@
     "@layerzerolabs/lz-definitions": "^2.3.27",
     "@layerzerolabs/lz-v2-utilities": "^2.3.27",
     "@layerzerolabs/scan-client": "^0.0.8",
-    "@maci-protocol/domainobjs": "0.0.0-ci.ef4f2d5",
-    "@maci-protocol/sdk": "0.0.0-ci.ef4f2d5",
+    "@maci-protocol/domainobjs": "0.0.0-ci.366681b",
```

Comment:
Just an open question: we could use `"@maci-protocol/domainobjs": "ci"` and always get the latest version of @maci-protocol. Should we do that in the future or is it safer to always fix a version? 🤔

Reply:
Didn't know you could do this! My take would be to keep an explicit version pinned, so it is simple to see what version we are using. Wdyt?